internal/pkg/podman/podman.go

```diff
@@ -224,6 +224,11 @@ func (hnd Handle) FindPrefixedContainer(prefixedName string) (iopodman.Container
 	return containers[0], nil
 }
 
+func (hnd Handle) VolumesPrune() error {
```

Comment:
this wrapper wants to have all the methods in the form VerbObject, thus it should be PruneVolumes.